cardano-cli/src/Cardano/CLI/EraBased/Options/Query.hs

```diff
@@ -364,14 +364,14 @@ pQueryDRepStateCmd era envCli = do
         <$> pSocketPath envCli
         <*> pConsensusModeParams
         <*> pNetworkId envCli
-        <*> pAllOrOnlyDRepHashSoure
-        <*> Opt.flag WithStake NoStake (mconcat
+        <*> pAllOrOnlyDRepHashSource
+        <*> Opt.flag NoStake WithStake (mconcat
```

Comment:
This is one of the two important bits of this PR: the arguments to [Opt.flag](https://hackage.haskell.org/package/optparse-applicative-0.18.1.0/docs/Options-Applicative.html#v:flag) are as follows:

1. The default value
2. The modified value

Here the two values were inverted. This change fixes that.